cves/CVE-2017-12616.yml

```diff
@@ -195,8 +194,9 @@ lessons:
     applies:
     note:
   secure_by_default:
-    applies:
-    note:
+    applies: true
+    note: |
+      should have preemptively implemented security measures in terms of requests.
```

Comment:
Could add more detail about what kind of security measures could be added relative to the requests(more authentication or more validation of input or all of the above)

Reply:
Added more detail.